src/lib/viewers/doc/DocControls.tsx

```diff
@@ -7,23 +7,30 @@ import FullscreenToggle, { Props as FullscreenToggleProps } from '../controls/fu
 import PageControls, { Props as PageControlsProps } from '../controls/page';
 import ThumbnailsToggle, { Props as ThumbnailsToggleProps } from '../controls/sidebar';
 import ZoomControls, { Props as ZoomControlsProps } from '../controls/zoom';
+import { AnnotationColor } from '../../AnnotationModule';
+import { AnnotationMode } from '../controls/annotations/types';
```

Comment:
We should centralize these types in an annotations-specific types file in a future PR.

Reply:
Will keep in mind.